src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -63,9 +64,17 @@ trait TestDriverComponent extends DriverComponent with DataAccess {
 
   def createTestSubmission(workspace: Workspace, methodConfig: MethodConfiguration, submissionEntity: Entity, rawlsUserRef: RawlsUserRef
                            , workflowEntities: Seq[Entity], inputResolutions: Map[Entity, Seq[SubmissionValidationValue]]
-                           , failedWorkflowEntities: Seq[Entity], failedInputResolutions: Map[Entity, Seq[SubmissionValidationValue]]) = {
+                           , failedWorkflowEntities: Seq[Entity], failedInputResolutions: Map[Entity, Seq[SubmissionValidationValue]]): Submission = {
+    createTestSubmission(workspace, methodConfig, submissionEntity, rawlsUserRef, workflowEntities, inputResolutions, failedWorkflowEntities, failedInputResolutions, WorkflowStatuses.Submitted)
+  }
+
+  def createTestSubmission(workspace: Workspace, methodConfig: MethodConfiguration, submissionEntity: Entity, rawlsUserRef: RawlsUserRef
+                           , workflowEntities: Seq[Entity], inputResolutions: Map[Entity, Seq[SubmissionValidationValue]]
+                           , failedWorkflowEntities: Seq[Entity], failedInputResolutions: Map[Entity, Seq[SubmissionValidationValue]],
+                           status: WorkflowStatus): Submission = {
     val workflows = (workflowEntities collect {
```

Comment:
These 2 can be simple `map`s, the first using a `val id: Option[String]` and only one `Workflow` constructor call